src/utils/getIndeterminateCircle.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+interface IndeterminateOption {
+  loading: boolean;
+}
+
+export default (options: IndeterminateOption) => {
+  if (!options.loading) {
+    return {
+      indeterminateStyleProps: {},
+      indeterminateStyleAnimation: null,
+    };
+  }
+
+  const animationName = 'circle-indeterminate-animate';
```

Comment:
Do not hard code this. Follow id generation logic to dynamic create this.

Reply:
@zombieJ 
Just added could you check please, and one more thing is when we are using useId hook inside Line component it performs another render which changes the transition-duration like in Circle, so that's why i updated the snapshot.

arfter this change i tested on demo and i see no impacts.